recipe/recipe.yaml

```diff
@@ -1,20 +1,22 @@
 context:
   name: stacrs
-  version: "0.5.5"
+  version: "0.5.8"
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
   url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/stacrs-${{ version }}.tar.gz
-  sha256: 22abb773dec5b73c11b81827d73675eb9683dc02b3a93a0e213a15f26d2d7275
+  sha256: 68ce670ec1d5eb20f40f41150bf47c77b265d7be35b81a1d36acb8f919de6bc4
 
 build:
   number: 0
   script:
+    - git config --system core.longpaths true # for windoze builds if we have patch dependencies
```

Comment:
Seems like a known issue - https://github.com/conda-forge/conda-smithy/issues/1484. Try the workaround used at https://github.com/conda-forge/staged-recipes/pull/29290/commits/d86e4bb10154fc3833cae054e197ea2d7119dac7 to set `CARGO_HOME`?